pkg/bootstrap/docker/up.go

```diff
@@ -45,6 +46,9 @@ const (
 	initialProjectDisplay = "My Project"
 	initialProjectDesc    = "Initial developer project"
 
+	defaultRedirectClient = "oauthclient/openshift-web-console"
```

Comment:
Hrm, yeah, that could result in bringing to `oc` packages that don't belong here (master.go for example imports a lot of server-side stuff).